src/index.ts

```diff
@@ -7,6 +7,7 @@ export {
   MarkdownView,
   MentionPopoverContent,
 } from './components';
+export { domainAndTitle, documentMetadata } from './helpers';
```

Comment:
Will these exports still be used in downstream projects after the `AnnotationDocumentInfo` refactor?

Reply:
In the end, I'm using it here, to determine if the `, ` needs to be added between the document title and the page number https://github.com/hypothesis/client/pull/7156/files#diff-efacdb37be7d29572b08b1ad918f880843257f82c18678597edd387aaa62f151R151-R158

It's a bit unfortunate not being able to move all the logic inside `AnnotationDocumentInfo`, but I think it might be possible to resolve this via CSS. I'll give it a thought.

Reply:
I found a way to do by adjusting a bit the markup. We don't need to expose `domainAndTitle` anymore.

`documentMetadata` is still needed in the annotation exporting process.